pkg/controllers/netpol/utils.go

```diff
@@ -61,3 +62,36 @@ func validateNodePortRange(nodePortOption string) (string, error) {
 	}
 	return fmt.Sprintf("%d:%d", port1, port2), nil
 }
+
+func getIPsFromPods(pods []podInfo, family api.IPFamily) []string {
+	var ips []string
+	for _, pod := range pods {
+		switch family {
+		case api.IPv4Protocol:
+			ip, _ := getPodIPv4Address(pod)
```

Comment:
Why are we ignoring the error here?

Reply:
Whoops, sorry, let me fix that!

However, if we ever get an error here, it will mean that some pod doesn't have an IP address (of the requested family). Which, I can imagine, sometimes might happen (pod waiting for IP address or maybe some Kubernetes issue that kube-router isn't really responsible for). I'm wondering how we should handle that error then. For now I think that we should just log it, but still proceed with collecting the other IPs - without early return.

@manuelbuil Thoughts?